components/automate-ui/habitat/plan.sh

```diff
@@ -37,41 +37,49 @@ do_unpack() {
   cp ${PLAN_CONTEXT}/environments/environment.prod.ts $CACHE_PATH/automate-ui/src/environments/environment.prod.ts
 }
 
+npm_install() {
+  # --unsafe-perm lets our package.json install task copy files when running
```

Comment:
Wow, that's a mouthful... Maybe clearer like this without losing any info...? 🤷‍♂  
```suggestion
  # --unsafe-perm lets our package.json install files when running
```
or this
```suggestion
  # --unsafe-perm lets our package.json install and copy files when running
```

Reply:
Just copied, but yeah we can change it.

Reply:
I'm not sure this is not getting "wronger" with the suggestions. There's a difference between "npm install" and "npm run install", and I'd suppose the latter was refered to by "install task".

Reply:
"our package.json install task" refers to a specific thing, and copying files is what this flag affects. 

How about something like 

> --unsafe-perm enables our package.json install task to copy files when running as superuser during the hab package building.

?

Reply:
Ah, thanks @phiggins. Yes, that added "to" helps for sure!